### DISCUSSION TITLE
2.1.5

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#11286] [General]
*  Resolve an issue introduced by 2.1.4 preventing older WSL1 distros from launching (solves #11255)
*  Display the missing optional component warning if vfpext is not installed

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.1.5'>2.1.5</a>.</em>

**Comment by muratyurdakul75** [comment]
Hi

I am using Windows Server 2022. Could the **autoMemoryReclaim=dropcache** setting in the .wslconfig file not be working on Windows Server? When using Windows 11, vmmem RAM usage was decreasing after a while. . However, after I started using Windows Server, vmmem RAM usage did not decrease.

Thanks

**wsl version : 2.0.14.0**

**.wslconfig setting**
[experimental]
autoMemoryReclaim=dropcache

**Comment by d2kx** [comment]
Is there a process in place for someone to have a look at the Microsoft Store distribution? Right now we have:

Dev Channel: WSL 2.1.4
Beta Channel: WSL 2.1.4
Release Preview Channel: WSL 2.0.15
Retail Channel (GA): WSL 2.1.5

I understand 2.1.5 was a hotfix and rushed to Stable, but it's annoying to be behind especially in the Release Preview Channel.

And before someone asks, no, the mechanisms do not work in the way that newer versions in the Retail Channel would overwrite your Insider offerings.

**Reply by alisonatwork** [reply]
Did you ever find a solution for this? I appear to be stuck on 2.0.15 as well after running `wsl --update` on the command line this morning, which promptly broke the previously-working mirrored networking setup from 23H2.

**Reply by d2kx** [reply]
I'll got Brandon LeBlanc to look into this next week hopefully, if you have a Twitter account feel free to reply to him that you're seeing the same behavior: https://twitter.com/brandonleblanc/status/1786861090766540847

**Comment by antgustech** [comment]
```
wsl --update --web-download
Checking for updates.
Updating Windows Subsystem for Linux to version: 2.1.5.
The handle is invalid.
Error code: Wsl/UpdatePackage/0x80070006
```
